code/API_definitions/geofencing-subscriptions.yaml

```
@@ -912,12 +913,14 @@ components:
       type: string
       description: |
         - NETWORK_TERMINATED - API server stopped sending notification
+        - SUBSCRIPTION_CREATION_ABORTED - Subscription cannot be created because the specified area cannot be managed (for asynchronous subscription creation)
```

If the reason is generic I would list the area case as an example, it may be others. Otherwise we should create a dedicated termination reason for it. If the client needs to know that the area is invalid we may need to inform of that specifically.

```suggestion
        - SUBSCRIPTION_CREATION_ABORTED - Subscription cannot be created (relevant for asynchronous subscription creation). For example, because the specified area cannot be managed 
```

---

In Commonalities we could use the following property:

```yaml
terminationDescription: 
   type: string 
```

---

https://github.com/camaraproject/Commonalities/blob/main/artifacts/camara-cloudevents/event-subscription-template.yaml#L629-L630